### PR TITLE
chore: update coins.json

### PR DESCRIPTION
**Pull request by szymonlesisz**
- update trezor-common submodule
- update Makefile: exclude "blockbook" field in favour of "blockchain_link" in coins build script
- update CoinInfo parsing code
- update supported coins
- add supported coins docs

### CODE REVIEW

File: docs/supported-coins.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Supported coins
+
+## The pipeline
+
+Do not change `src/data/coins.json` manually.
+
+The one and only source of truth are `*.json` definitions declared and maintained in the [firmware repository](https://github.com/trezor/trezor-firmware/tree/master/common/defs). 
+
+These are duplicated as read-only in the [trezor-common](https://github.com/trezor/trezor-common) repository.
```
Comment (tsusanka) [inline docs/supported-coins.md]: Maybe better:

```suggestion
These are exported to a read-only [trezor-common](https://github.com/trezor/trezor-common) repository.
```

File: docs/supported-coins.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+`trezor-common` is included in `trezor-connect` as git submodule mounted at `submodules/trezor-common`.
+
+## Update and maintenance in trezor-connect
+
+_Make sure that desired `[coin].json` definition is present in `trezor-firmware` repository *and* corresponding [support for connect](https://github.com/trezor/trezor-firmware/blob/master/common/defs/support.json#L3) is enabled._
```
Comment (tsusanka) [inline docs/supported-coins.md]: Small preference to replace https://github.com/trezor/trezor-firmware/blob/master/common/defs/support.json#L3 with https://github.com/trezor/trezor-firmware/blob/4e005de02fbb9db11a304587ec1abe8aab80cdfd/common/defs/support.json#L3 . (You get such link by pressing `y` btw.) This way the link will always link to the same content

File: docs/supported-coins.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+update `trezor-common` submodule
```
Comment (tsusanka) [inline docs/supported-coins.md]: Now I am just bikeshedding..

```suggestion
1. Update `trezor-common` submodule:
```

File: docs/supported-coins.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+```
+make submodule
+```
+build `src/data/coins.json` file using `trezor-common/cointool`
```
Comment (tsusanka) [inline docs/supported-coins.md]:
```suggestion
2. Build `src/data/coins.json` file using `trezor-common/cointool`:
```

**Review by tsusanka — APPROVED** [review]
Approve for docs.

File: docs/supported-coins.md (hunk 4 above)
Comment (szymonlesisz) [reply docs/supported-coins.md]: 239c7de

File: docs/supported-coins.md (hunk 3 above)
Comment (szymonlesisz) [reply docs/supported-coins.md]: 239c7de

File: docs/supported-coins.md (hunk 2 above)
Comment (szymonlesisz) [reply docs/supported-coins.md]: 239c7de

File: docs/supported-coins.md (hunk 1 above)
Comment (szymonlesisz) [reply docs/supported-coins.md]: 239c7de

**Review by tsusanka — APPROVED** [review]

**Review by goodhoko — APPROVED** [review]
LGTM. Good we get rid of the field patching imperative logic. .)

**Comment by szymonlesisz** [comment]
we didnt get rid of it, it was changed by [trezor-common](https://github.com/trezor/trezor-firmware/commit/ac658677717a69fcb71de3f2802ada7cc0a31142)

**Pull request merged**